packages/core/src/util/hook.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const iitmHook = require('./iitmHook');
+const requireHook = require('./requireHook');
+const isESMApp = require('./esm').isESMApp;
+
+/*
+ * pureEsmLibraries
+ *
+ * This array lists all the libraries that are native ECMAScript Modules.
+ */
+const pureEsmLibraries = ['square-calc'];
```

Comment:
This needs to be changed as well. This is no longer correct.

Reply:
The best approach I guess is:

`hook.onModuleLoad('square-calc', { nativeEsm: true})`

Reply:
Updated the code